### PR TITLE
feat(hooks): adapt to POEditor's change in hooks management

### PR DESCRIPTION
**Pull request by p-j**
POEditor has changed the way webhooks work.
This is more a workaround than a proper solution.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,11 +84,11 @@ const {
...
   updateTranslations,
-  viewProject,
-} = require('gitpo')
+  viewProject
+} = require("gitpo");
 
-const languages = await listProjectLanguages(projectId)
-console.log(languages)
+const languages = await listProjectLanguages(projectId);
+console.log(languages);
```
Comment (albanv) [inline README.md:91]: forgotten ?

File: cli.js [hunk 2]
```diff
@@ -132,20 +132,32 @@ inquirer
...
+          })
         break
       case PROJECT_CLEAN:
         cleanTranslationJSON(file, override)
           .then(() => console.log('File ready'))
-          .catch(() => console.log('Something went wrong'))
+          .catch((err) => {
+            console.log('Something went wrong')
+            console.error(err)
+          })
```
Comment (albanv) [inline cli.js]: you should refactor it within a `handleError` function

File: lib/webhooks/updateTranslations.js [hunk 3]
```diff
@@ -1,11 +1,4 @@
...
-module.exports = ({ API_TOKEN }) => (projectId, languages) =>
-  Promise.all(
-    languages.map((lng) =>
-      rp({
-        method: 'GET',
-        url: `https://poeditor.com/api/webhooks/github?api_token=${API_TOKEN}&id_project=${projectId}&language=${lng}&operation=export_terms_and_translations`,
-      })
-    )
-  )
+module.exports = ({ HOOKS }) => (projectId, languages) =>
```
Comment (albanv) [inline lib/webhooks/updateTranslations.js]:
```suggestion
module.exports = ({ HOOKS }) => async (projectId, languages) =>
```

File: lib/webhooks/importNewTerms.js [hunk 4]
```diff
@@ -1,7 +1,3 @@
 const rp = require('request-promise')
 
-module.exports = ({ API_TOKEN }) => (projectId) =>
-  rp({
-    method: 'GET',
-    url: `https://poeditor.com/api/webhooks/github?api_token=${API_TOKEN}&id_project=${projectId}&language=en&operation=get_terms`,
-  })
+module.exports = ({ HOOKS }) => (projectId) => rp({ method: 'GET', url: HOOKS[projectId]['IMPORT'] })
```
Comment (albanv) [inline lib/webhooks/importNewTerms.js]:
```suggestion
module.exports = ({ HOOKS }) => async (projectId) => rp({ method: 'GET', url: HOOKS[projectId]['IMPORT'] })
```

**Review by albanv — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (p-j) [reply README.md:91]: That's the `README`

File: cli.js (hunk 2 above)
Comment (p-j) [reply cli.js]: good point

File: README.md (hunk 1 above)
Comment (albanv) [reply README.md:91]: OH 😅

**Review by albanv — APPROVED** [review]

**Pull request merged**